lisa/sut_orchestrator/libvirt/ch_platform.py

```diff
@@ -65,6 +65,9 @@ def _configure_node(
         else:
             node_context.firmware_path = node_runbook.firmware
 
+        if node_runbook.disk_size_in_gb:
+            node_context.os_disk_size_in_gb = node_runbook.disk_size_in_gb
+
```

Comment:
This has to be present in `_configure_node` of `BaseLibvirtPlatform` class. That way disk_size can be used with qemu platform as well.